### PR TITLE
Casey Fazio

### PR DESCRIPTION
**Pull request by casefaz**

### CODE REVIEW

File: final_prep/README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ Take your time as you go back and refactor your exercises from each section. We'
 
 ## Gearing Up for Gear Ups
 
-Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
+Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry.
 
 - [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
+  - [ ] [⚡️Casey's Gear Up Assigment⚡️](https://gist.github.com/7d103ad1d5252288c27aac367d7faf6b.git)
```
Comment (philmccarthy) [inline final_prep/README.md:35]: Excellent reflections, Casey. 100% with you on the societal scarcity mentality and how it manifests itself in our identities and actions. Your perspective is valued in these conversations at Turing.

File: final_prep/README.md [hunk 2]
```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1204" alt="Screen Shot 2022-02-12 at 1 17 58 PM" src="https://user-images.githubusercontent.com/98674727/153728804-0763b70e-9c59-462f-8e9f-701d4a08d075.png">
+- `Add Week 2 Screenshot Here`<img width="1209" alt="Screen Shot 2022-02-12 at 1 19 23 PM" src="https://user-images.githubusercontent.com/98674727/153728814-adc440d2-20e6-4c52-94b1-c29ab14118a7.png">
+- `Add Week 3 Screenshot Here` <img width="1211" alt="Screen Shot 2022-02-12 at 1 19 36 PM" src="https://user-images.githubusercontent.com/98674727/153728829-51220745-119c-493a-a454-9283b150afc0.png">
```
Comment (philmccarthy) [inline final_prep/README.md:49]: Calendar is looking great. I know the Mod 1 calendar hasn't been fully updated yet but when it does, highly recommend specifying some of your work blocks (ie, what will you study based on the lessons that week?).

Big fan of how you're keeping `zone out mindless scroll time` on the schedule, but timeboxing it too.

File: final_prep/README.md [hunk 3]
```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
...
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Casey's Mentee DTR](https://gist.github.com/casefaz/5f5ea03e742987f53240da413b2558c0)
```
Comment (philmccarthy) [inline final_prep/README.md:57]: Nice work on your mentorship prep! Definitely refer back to this gist when you get introduced / meet your mentor in a few weeks.

File: final_prep/README.md [hunk 4]
```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
...
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: [Casey's pre-teaching exercise](https://gist.github.com/casefaz/8d7a1fd68c23236a36ca9a331f705964)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: [Collaboration and Working-Style Preferences](https://gist.github.com/casefaz/f9b1493509eba5091501719e533e1310)
```
Comment (philmccarthy) [inline final_prep/README.md:72]: Good work! I encourage you to build on this group work DTR throughout Turing. We use it every time we kick off a team project, so you can 1) adjust as you learn how you work best in this new, technical space, and 2) save yourself the effort of filling out a new DTR every time

File: section1/reflection.md [hunk 5]
```diff
@@ -1,19 +1,33 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I really appreciated the various approaches to asking questions. I find that I tend to ask as soon as I have a questions, and feel that I would benefit from troubleshooting and saying the questions out loud, wording it very specifically with examples before finally pulling the trigger to ask. I also ordered a "rubber ducky" for out loud problem solving.
```
Comment (philmccarthy) [inline section1/reflection.md:4]: Ducks love code

File: section1/reflection.md [hunk 6]
```diff
@@ -1,19 +1,33 @@
...
+* An interpolation is sticking data in the middle of a string
+
+`puts "#{animal} mussel is an invasive species found in Eurasia found to harm native species of that area."`
 
 8. What method is used to get input from a user?
+* `gets.chomp`
 
 9. Name and describe two common string methods:
+* .reverse - prints the string in reverse
+* .length - prints out the number of characters in the string
```
Comment (philmccarthy) [inline section1/reflection.md:33]: good examples!

File: section2/exercises/ex29.rb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+if people == dogs
+  puts "People are dogs."
+end
+
+# Q: What do you think the if does to the code under it?
+# A: the if tells the code to print if the statement is true
+# Q: Why does the code under the if need to be indented two spaces?
+# A: for code readability/to tell other programmers it's a code block
+# Q: What happens if it isn't indented?
+# A: it looks messy/is hard to read
```
Comment (philmccarthy) [inline section2/exercises/ex29.rb:40]: +1. Totally works without proper style but doesn't demonstrate developer empathy 🙂

File: section2/exercises/ex30.rb [hunk 8]
```diff
@@ -0,0 +1,56 @@
+#assigns value to people
```
Comment (philmccarthy) [inline section2/exercises/ex30.rb:1]: minor style note: add whitespace between the `# Beginning of comment`. A little easier to parse the comments that way

File: section2/exercises/ex31.rb [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+
+print "> "
+option = $stdin.gets.chomp
+
+if option == "1"
+  puts "It's clear you've entered a witches cottage. They appear to be out at the moment, but the cauldron is simmering, dry herbs are hanging on the wall and a raven is perched above a set of appendages marinating in glass jars. What do you do?"
+  puts "1. Talk to the raven."
+  puts "2. Try to use the materials to cast a spell of your own."
+  puts "3. Wreak havoc on the small cabin."
```
Comment (philmccarthy) [inline section2/exercises/ex31.rb:52]: Excellent little program you wrote here!

File: section2/exercises/methods.rb [hunk 10]
```diff
@@ -12,21 +12,30 @@ def print_name
...
-# Write a method that takes in two strings as arguments and prints 
-# a concatenation of those two strings. Example: The arguments could be 
-# (man, woman) and the end result might output: "When Harry Met Sally".  
-# Then call your method three times with different arguments passed in. 
+# Write a method that takes in two strings as arguments and prints
+# a concatenation of those two strings. Example: The arguments could be
+# (man, woman) and the end result might output: "When Harry Met Sally".
+# Then call your method three times with different arguments passed in.
 
+sentence = ["Felt needling", "is cool."]
```
Comment (philmccarthy) [inline section2/exercises/methods.rb:34]: See above- how could we define a method to accomplish the same thing that you figured out with Array methods?

File: section3/exercises/arrays.rb [hunk 11]
```diff
@@ -17,51 +17,59 @@
...
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
-
+foods.push("broccoli")
+p foods
 
 # YOU DO: Write code below that removes the last item of food from the foods array and
 # print the changed array to verify that item has been removed
+foods.pop()
```
Comment (philmccarthy) [inline section3/exercises/arrays.rb:60]: In Ruby, we generally won't use empty parens `()` on a method call without any arguments. So we can just say `foods.pop`. Technically works either way though!

File: section3/reflection.md [hunk 12]
```diff
@@ -1,17 +1,37 @@
...
+food: 1000,
+toys: 1000000000,
+good_animals: 400,
+bad_animals: 0
+}
+```
+
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
 
+`states.values[1]`
```
Comment (philmccarthy) [inline section3/reflection.md:26]: This works! Can you think of another, more direct way to access the value associated with a certain key in a hash?

**Review by philmccarthy — APPROVED** [review]
Hi @casefaz great work with the project. No changes are needed.

I left a few comments throughout, feel free to ask any questions that come up in the replies.

I appreciate how you incorporated my feedback over the last couple weeks. You're well prepared for success at Turing. Good luck in Mod 1!

**Comment by casefaz** [comment]
Seriously appreciate the feedback yet again. I will look over the
recommendations and hope to continue to get to hear from you throughout
Turing!!

**Pull request closed without merging**